images/releaser/release.sh

```diff
@@ -21,3 +21,5 @@ echo $repo
 oras tag docker.io/"$1" $digest
```

Comment:
Whole script was a placeholder rather, this will have to be changed before it's ran on the real images. For last two releases I've used something like this:
```
#!/usr/bin/env bash

TAG="v2.0.0"
REGISTRY="docker.io"
REPO="nephio"
#network-config-operator resource-backend-controller
#v2.0.x:
for image in kpt-backstage-plugins oai-ran-controller porch-server porch-controllers porch-function-runner porch-wrapper-server nephio-operator vlan-fn nfdeploy-fn nad-fn ipam-fn interface-fn gen-configmap-fn dnn-fn configinject-fn ueransim-deploy-fn ; do
#v1.0.x:
#for image in network-config-operator resource-backend-controller kpt-backstage-plugins nephio-operator vlan-fn amf-deploy-fn smf-deploy-fn upf-deploy-fn ipam-fn ueransim-deploy-fn configinject-fn interface-fn dnn-fn nad-fn free5gc-operator ; do

echo "---------------------------$image------------------------------"
#trivy image --format template --template "@/home/ubuntu/dev/trivy.tpl" -o reports/report_$i.html docker.io/nephio/$i:v1.0.0
URI="$REGISTRY"/"$REPO"/"$image":"$TAG"
echo "Tags in "$REGISTRY"/"$REPO"/"$image":"
crane ls "$REGISTRY"/"$REPO"/"$image"
DIGEST_RAW=$(crane digest "$URI")
DIGEST=$(echo "${DIGEST_RAW//:/-}")
echo "$URI digest: $DIGEST"
echo "Tagging $URI with $DIGEST"
#crane tag "$URI" "$DIGEST"
echo "Grype vulnerability scan"
grype -v "$URI" | tee reports/"$image"_"$TAG"_grype_report.txt
echo "Generating SBOM:"
syft "$URI" --output=spdx-json >> sboms/"$image"_"$TAG".sbom
echo "Pushing SBOM to registry"
oras push --artifact-type artifact/sbom "$REGISTRY"/"$REPO"/"$image":"$DIGEST".sbom sboms/"$image"_"$TAG".sbom:application/json
#oras push --artifact-type artifact/sbom "$REGISTRY"/"$REPO"/"$image":"$TAG".sbom sboms/"$image"_"$TAG".sbom:application/json
echo "Pushing vulnearability scan results"
oras push --artifact-type artifact/doc "$REGISTRY"/"$REPO"/"$image":"$DIGEST"_scan.txt reports/"$image"_"$TAG"_grype_report.txt:text/plain
#oras push --artifact-type artifact/doc "$REGISTRY"/"$REPO"/"$image":"$TAG"_scan.txt reports/"$image"_"$TAG"_grype_report.txt:text/plain

echo "Signing image"
cosign sign --key cosign_prod/cosign.key "$REGISTRY"/"$REPO"/"$image"@"$DIGEST_RAW" -y
echo "Listing repo content"
crane ls "$REGISTRY"/"$REPO"/"$image"
echo "---------------------------$image------------------------------"
done
```
But maybe for now just remove all functionality just leave image signing.